docs/xnat2bids.md

```diff
@@ -18,7 +18,7 @@ $ xnat2bids [OPTIONS] SESSION BIDS_ROOT_DIR
 * `-u, --user TEXT`: XNAT User
 * `-p, --pass TEXT`: XNAT Password
 * `-h, --host TEXT`: XNAT'sURL  [default: https://xnat.bnc.brown.edu]
-* `-S, --session-suffix TEXT`: Suffix of the session for BIDS defaults to 01.              This will produce a session label of sess-01.              You likely only need to change the default for multi-session studies  [default: 01]
+* `-S, --session-suffix TEXT`: Suffix of the session for BIDS defaults to 01.              This will produce a session label of sess-01.              You likely only need to change the default for multi-session studies  [default: -1]
```

Comment:
darn, these are auto generated and it means that the docs say the default is -1. Maybe we should clarify in the comment